### PR TITLE
build: update zone.js to 0.11.3

### PR DESCRIPTION
**Pull request by JiaLiPassion**
Update zone.js to the latest version of 0.11.3

### CODE REVIEW

**Review by alan-agius4 — APPROVED** [review]
LGTM. Thanks.

File: integration/angular_cli/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "@angular/router": "~9.1.1",
     "rxjs": "~6.5.4",
     "tslib": "^1.10.0",
-    "zone.js": "~0.10.2"
```
Comment (clydin) [inline integration/angular_cli/package.json:25]: This test project is still using Angular v9 which I believe doesn't support zone.js 0.11?

**Review by clydin — CHANGES_REQUESTED** [review]
Can you split this into two separate commits so that the changelog will show that new projects use zone.js 0.11.x?

1) The current build commit
2) Changes to `packages/schematics/angular/utility/latest-versions.ts` in a `feat(@schematics/angular): ...` commit

File: integration/angular_cli/package.json (hunk 1 above)
Comment (JiaLiPassion) [reply integration/angular_cli/package.json:25]: I believe zone.js 0.11 still supports Angular v9, there should no breaking changes impact Angular v9.

**Comment by JiaLiPassion** [comment]
@clydin , got it, I split the PR into two commits, thanks.

**Review by clydin — APPROVED** [review]

**Pull request merged**